openslides/agenda/static/templates/agenda/item-import.html

```diff
@@ -23,71 +23,121 @@ <h3 translate>Import by copy/paste</h3>
+      <i>{{ importcounter }} / {{ titleitems.length }} {{ "imported" | translate }}</i>
     </progressbar>
   </div>
 </div>
+<div class="spacer">
+  <a ng-if="importcounter > 0 && importcounter == titleitems.length" ui-sref="agenda.item.list"
+      class="btn btn-default">
+    <i class="fa fa-angle-double-left fa-lg"></i>
+    <translate>Back to agenda</translate>
+  </a>
+</div>
 
 <hr>
 
-<h3 translate>Import by CSV file</h3>
-<p translate>Select a CSV file to import agenda items!
+<h2 translate>Import by CSV file</h2>
+  <div class="block row">
+    <div class="title">
+      <h3 translate>Select a CSV file
+    </div>
+    <div class="block right import">
+      <label class="label" for="inputSeparator" translate>Separator</label>
+      <input type="text" ng-model="separator" ng-change="setSeparator()" ng-init="separator=separator" id="inputSeparator">
+      <br>
+      <label class="label" for="selectEncoding" translate>Encoding</label>
+      <select ng-model="encoding" ng-options="enc as enc for enc in encodingOptions"
+          ng-selected="setEncoding()" ng-init="encoding=encoding" id="selectEncoding"></select>
+      <ng-csv-import
+          content="csv.content"
+          header="csv.header"
+          header-visible="csv.headerVisible"
+          separator="csv.separator"
+          separator-visible="csv.separatorVisible"
+          result="csv.result"
+          encoding="csv.encoding"
+          encoding-visible="csv.encodingVisible"></ng-csv-import>
+    </div>
+  </div>
 
-<p translate>Please note:</p>
-<ul><!--TODO: utf-8 encoding still required with angular-csv? -->
-    <li><translate>Required comma separated values</translate>:<br>
-        <code translate>'title, text'</code>
-    <li translate>Text and duration are optional and may be empty.
-    <li translate>The header in first line is required.
-    <li translate>Required CSV file encoding is UTF-8.
-    <li><a href="https://github.com/OpenSlides/OpenSlides/wiki/CSV-Import" translate>
-          Use the CSV example file from OpenSlides Wiki.
-        </a>
-</ul>
+  <h4 translate>Please note:</h4>
+  <ul>
+      <li><translate>Required comma or semicolon separated values with these column header names in the first row</translate>:<br>
+          <code>'title, text'</code>
+      <li translate>Text is optional and may be empty.
+      <li translate>Only double quotes are accepted as text delimiter (no single quotes).
+      <li><a href="https://github.com/OpenSlides/OpenSlides/wiki/CSV-Import" translate>
+            Use the CSV example file from OpenSlides Wiki.
+          </a>
+  </ul>
 
-<ng-csv-import
-  content="csv.content"
-  class="import"
-  header="csv.header"
-  separator="csv.separator"
-  result="csv.result"></ng-csv-import>
+  <div ng-if="csv.result">
+    <h3 translate>Preview</h3>
+    <table class="table table-striped table-bordered table-condensed">
+      <thead>
+        <tr>
+          <th>
+          <th>#
+          <th translate>Title
```

Comment:
Are you sure that you can skip the closing tag? I thought is was needed for the translation